cdc/sink/mq/mq.go

```diff
@@ -342,6 +338,9 @@ func (k *mqSink) EmitDDLEvent(ctx context.Context, ddl *model.DDLEvent) error {
 
 // Close the producer asynchronously, does not care closed successfully or not.
 func (k *mqSink) Close(ctx context.Context) error {
+	k.flushWorker.close()
```

Comment:
will it happens that mqSink.Close() has been called while mq.bgFlushTs() being called in a separate goroutine?

Reply:
I don't think there will be such a problem because the global sink close is the processor's responsibility, and the processor stops all table pipelines before shutting down the sink.

Reply:
In my opinion, `bgFlushTs` will not be affected by the table pipeline. Normally, it exits only after ctx.done().
Maybe Sink needs to maintain a ctx and cancel.